authorization/client/userDevices.go

```diff
@@ -169,6 +169,19 @@ func (d *UserDevicesManager) UpdateUserDevices(ctx context.Context, userID strin
 	return userDevices[userID], nil
 }
 
+func (d *UserDevicesManager) ChecksUserDevices(userID string, deviceIDs []string) bool {
```

Comment:
Just came to my mind, should we really name it User? Isn't it the owner? Also RPC calls, AddUserDevices. It's assigning to the owner, and the user doesn't have to necessarily be an end-user, or? @Danielius1922 , @jkralik what do you think?

Reply:
`Owner` seems better, I think @jkralik suggested something similar: renaming GetUserDevices to GetOwnerDevices. If it better expresses what it does I think we should rename it, though we should rename it in all places and not just in one or two.